subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/r22/ClientShutdownCrossVersionSpec.groovy

```diff
@@ -41,12 +39,13 @@ class ClientShutdownCrossVersionSpec extends ToolingApiSpecification {
 
     def "can shutdown tooling API session when no operations have been executed"() {
         given:
-        toolingApi.close();
+        toolingApi.close()
 
         when:
         toolingApi.withConnection {}
         then:
         IllegalStateException e = thrown()
+        e
```

Comment:
What are we checking here? We should change the previous line to do the following:

    def t = thrown(IllegalStateException)

And then on this line check the correct exception message via `t.message = '...'`.